FirebaseAdmin/FirebaseAdmin/Util/HttpUtils.cs

```diff
@@ -29,6 +30,17 @@ internal sealed class HttpUtils
 
         private HttpUtils() { }
 
+        public static string FormatString(string str, Dictionary<string, string> urlParams)
```

Comment:
A format string method on a http utillity doesn't really explain what its doing right? I feel a name like `StringUtils.ReplacePlaceholders(...)` would make more sense.

Reply:
I think that it is good suggestion, I make StringUtils class and define ReplacePlaceholders() function.